modules/ts-capi-sync-list-duplicate-resources.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cluster_api_machine_management/cluster-api-troubleshooting.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ts-capi-sync-list-duplicate-resources_{context}"]
+= Duplicated machine set and machine resources
+
+On clusters that support migrating Machine API resources to Cluster API resources, some resources seem to have duplicate instances in the output of {oc-first} commands that list resources and in the {product-title} web console.
+
+Cause::
+
+When you install an {product-title} cluster that uses the default configuration options, the installation program provisions the following infrastructure resources in the `openshift-machine-api` namespace:
++
+--
+* One control plane machine set that manages three control plane machines.
+* One or more compute machine sets that manage three compute machines.
+* One machine health check that manages spot instances.
+* Compute machines that are created according to the compute machine set specifications.
+--
++
+On clusters that support migrating Machine API resources to Cluster API resources, a two-way synchronization controller creates the following Cluster API resources in the `openshift-cluster-api` namespace:
++
+--
+* One cluster resource.
+* One provider-specific infrastructure cluster resource.
+* One or more machine templates that correspond to compute machine sets.
+* One or more compute machine sets that manage three compute machines.
+* Compute machines that are created according to the machine template and compute machine set specifications.
+* Infrastructure machines that correspond to compute machines.
+--
++
+These Cluster API resources have the same names as their counterparts in the `openshift-machine-api` namespace.
+
+Consequence::
+
+Due to this behavior, instances of machine set and machine resources that seem to be duplicates appear in the output of `oc` commands that list resources and in the {product-title} web console.
+
+Workaround::
+
+Although the resources have the same names as their counterparts in the other namespace, only the resources that use the current authoritative API are active.
+The synchronization controller creates and maintains the corresponding resources that do not use the current authoritative API in an unprovisioned (`Paused`) state to prevent unintended reconciliation.
+
+Result::
+
+Only one of each resource that seems to be a duplicate is active at a time.
+The inactive nonauthoritative resources do not impact functionality.
++
+[IMPORTANT]
+====
+Do not delete any nonauthoritative resource that does not use the current authoritative API unless you want to delete the corresponding resource that does use the current authoritative API.
+
+When you delete a nonauthoritative resource that does not use the current authoritative API, the synchronization controller deletes the corresponding resource that does use the current authoritative API.
```

Comment:
If I am reading this right, we don't need the first sentence. Because, according to the second sentence, the authoritative resource is automatically deleted when you delete the nonauthoritative resource.